proto/event.proto

```diff
@@ -52,3 +52,117 @@ enum SlotStatus {
   // The highest slot that has been voted on by supermajority of the cluster, ie. is confirmed.
   Confirmed = 2;
 }
+
+// MessageHeader
+message MessageHeader {
+  uint32 num_required_signatures = 1;
+  uint32 num_readonly_signed_accounts = 2;
+  uint32 num_readonly_unsigned_accounts = 3;
+}
+
+// CompiledInstruction
+message CompiledInstruction {
+  uint32 program_id_index = 1;
+  repeated uint32 accounts = 2;
+  bytes data = 3;
+}
+
+message LoadedAddresses {
+  repeated bytes writable = 1;
+  repeated bytes readonly = 2;
+}
+
+message MessageAddressTableLookup {
+  bytes account_key = 1;
+  repeated uint32 writable_indexes = 2;
+  repeated uint32 readonly_indexes = 3;
+}
+
+message V0Message {
+  MessageHeader header = 1;
+  repeated bytes account_keys = 2;
+  bytes recent_block_hash = 3;
+  repeated CompiledInstruction instructions = 4;
+  repeated MessageAddressTableLookup address_table_lookup = 5;
+}
+
+message V0LoadedMessage {
+  V0Message message_ = 1;
+  LoadedAddresses loaded_adresses = 2;
+}
+
+message LegacyMessage {
+  MessageHeader header = 1;
+  repeated bytes account_keys = 2;
+  bytes recent_block_hash = 3;
+  repeated CompiledInstruction instructions = 4;
+}
+
+message SanitizedMessage {
+  oneof message_payload {
+    LegacyMessage legacy = 1;
+    V0LoadedMessage v0 = 2;
+  }
+}
+
+message SanitizedTransaction {
+  SanitizedMessage message_ = 1;
+  bytes message_hash = 2;
+  bool is_simple_vote_transaction = 3;
+  repeated bytes signatures = 4;
+}
+
+message InnerInstruction {
+  uint32 index = 1;
+  repeated CompiledInstruction instructions = 2;
+}
+
+// F64 is like google.protobuf.DoubleValue, but avoid importing another proto.
```

Comment:
Importing `google.protobuf.DoubleValue` is fine. Looks like it is bundled with `prost`: https://github.com/tokio-rs/prost/blob/fc9fbd9823f93774004975f7794dac112a2ff4ef/prost-build/third-party/include/google/protobuf/wrappers.proto

Reply:
> Importing `google.protobuf.DoubleValue` is fine. Looks like it is bundled with `prost`: https://github.com/tokio-rs/prost/blob/fc9fbd9823f93774004975f7794dac112a2ff4ef/prost-build/third-party/include/google/protobuf/wrappers.proto

Thanks. Updated with `google.protobuf.DoubleValue`